modules/docs/src/03-protocols/04-deriving-cli.md

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
I'd move this file under `modules/docs/src/03-protocols/simple-rest-json/04-deriving-cli.md `

so it is along the other simpleRestJson things

Reply:
![Screen Shot 2022-08-19 at 09 15 11](https://user-images.githubusercontent.com/5230460/185626547-3dfe76f3-08b4-46d4-828c-bbbf5fa240d4.png)

Reply:
Why ?

Reply:
my understanding is that deriving clis works with this particular protocol, maybe I'm wrong though

Reply:
no , deriving cli's is protocol agnostic

Reply:
it may not be tied to simpleRestJson, but it sure seems to be built on http & json

Reply:
Perhaps you can elaborate in standup. 
Currently IIUC , if there are http based traits that information is used for documentation . however its not a requirement.
json is a requirement iff your data is structured in a way that needs a better mechanism then positional or named arguments, this doesn't affect how or if the data is transmitted at all over the network or otherwise using any particular encoding